### PR TITLE
Awesome Angular GraphQL files

### PR DESCRIPTION
**Pull request by WickedBrat**
In this PR:
- Created the file README, CONTRIBUTE and LICENSE
- Updated with the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,105 @@
...
+  - [Basic Angular Apollo Boilerplate](https://github.com/WickedBrat/angular-hasura-boilerplate/tree/master/basic)
+  - [Advanced Angular Apollo Boilerplate](https://github.com/WickedBrat/angular-hasura-boilerplate/tree/master/advanced)
+
+## GraphQL BaaS Providers
+
+> Get a GraphQL backend for your applications using these service providers
+
+- [Graphcool](https://www.graph.cool): Self-Hosted GraphQL BaaS
+- [Hasura](https://hasura.io): Instant GraphQL APIs on Postgres
+- [Scaphold](https://scaphold.io): GraphQL Backend As A Service
```
Comment (praveenweb) [inline README.md]: Scaphold is no longer active. So let's remove that.

File: README.md [hunk 2]
```diff
@@ -1 +1,105 @@
...
+> Boilerplates and examples for a headstart in development
+
+- Angular Apollo GraphQL
+  - [Angular Fullstack Graphql](https://github.com/graphql-boilerplates/angular-fullstack-graphql)
+  - [Angular Todo App using Apollo Client](https://github.com/WickedBrat/hasura-angular-todo)
+  - [Angular AppSync Starter App](https://github.com/aws-samples/aws-mobile-appsync-chat-starter-angular)An AWS AppSync Chat Starter App written in Angular.
+  - [Next.js With Apollo](https://github.com/zeit/next.js/tree/master/examples/with-apollo) - Boilerplate for building Next.js apps with Apollo.
+  - [Apollo Angular SSR](https://github.com/kamilkisiela/apollo-angular-ssr) - Apollo-Angular with SSR and Store Rehydration
+  - [Apollo Angular](https://github.com/apollographql/apollo-angular)
+  - [Hello World Angular Apollo Boilerplate](https://github.com/WickedBrat/angular-hasura-boilerplate/tree/master/hello-world)
```
[the remark this answers is not in the document]
Comment (WickedBrat) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -1 +1,103 @@
...
+- [GraphQL vs Firebase](https://www.prisma.io/blog/graphql-vs-firebase-496498546142/)
+## Stories
+
+> Good and bad experiences
+
+- [Jeff Cross: Faster Modern Apps with Angular and GraphQL](https://www.graphql.com/articles/angular-graphql-faster-modern-apps)
+- [(Video) Lessons from 4 Years of GraphQL](https://www.youtube.com/watch?v=zVNrqo9XGOs)
+- [2017: The year in GraphQL](https://dev-blog.apollodata.com/2017-the-year-in-graphql-124a050d04c6)
+- [Harry Wolf: Experiences with GraphQL](http://hswolff.com/blog/experiences-with-graphql/)
+- [JetRuby: Our experience with GraphQL](https://expertise.jetruby.com/our-experience-with-node-js-react-graphql-c40007ad4373)
```
Comment (praveenweb) [inline README.md]: This article seems to be focussed on react.

Comment (WickedBrat) [reply README.md]: @praveenweb PTAL

**Pull request merged**